src/documentation/chunk.rs

```diff
@@ -204,6 +205,13 @@ impl CheckableChunk {
     pub fn last_span(&self) -> Option<(&Range, &Span)> {
         self.source_mapping.get_index(self.source_mapping.len() - 1)
     }
+
+    pub fn complete_coverage(&self) -> Result<((&Range, &Span), (&Range, &Span))> {
```

Comment:
`/// I like docs :)`

Reply:
I would expect this to return `(Range, Span)` where `Span { start: first.start, end: last.end } ` and `first_range.start..last_range.end`

Reply:
After a week, I can't remember what I was thinking when I implemented this in the first place...

Reply:
#verygoodvacays ;)

I tend to overcomment because of that, sometimes every block of code has a line to declare the intent and outcome of it, especially when dealing with nested iterators